Update UG for tasks section (#164)
AY2223S1-CS2103T-T10-1/tp: docs/UserGuide.md

#### 2.2.1. Add task
You may add a task using the `add-task` command.

<div markdown="span" class="alert alert-info">:information_source: **Note:** 

> Make sure you have [added a module](#211-add-module) to Plannit before 
> proceeding!.

Reviewer: ```suggestion
Make sure you have [added a module](#211-add-module) to Plannit before 
proceeding!
```
Noticed additional fullstops after !. Other than that good use of active voice!

Author: Good catch!

</div>

This command will require two flags:

| Field                            | Flag  | Constraints                                             |
|----------------------------------|-------|---------------------------------------------------------|
| **Module Code**                  | `m/`  | Can only be non-empty string of alphanumeric characters |
| **Task Description of New Task** | `td/` | Can be any string of characters                         |

Format: `add-task m/MODULE_CODE td/TASK_DESCRIPTION`

> * Each task **must** belong to a specific module.

Reviewer: Taken from PR by @cheeheng: https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/159 for reference
Code excerpt:
`Module code must correspond to currently displayed module on screen (case-insensitive)`

Notice that usage of must on whether it is bolded is inconsistent. Hence, I feel that we should come to a general consensus on whether it should be bolded.

@AY2223S1-CS2103T-T10-1/developers

Author: I shall remove my bold, since it seems unnecessary.

add-task m/CS2103T td/Complete tP tasks for W07
```
In the above example, we are adding the task `Complete tP tasks for W07` to the
module with module code `CS2103T`. Here's a screenshot of the command before

> and after executing:

Reviewer: ```suggestion
and after execution:
```

Felt that it sounds better this way .-. just a suggestion

Author: Indeed, I will make the changes!

Author: After some consideration, I realised my sentence did not make sense, since I was not providing a screenshot of the _command_, but rather a screenshot of _Plannit_. I will hence be changing it to 

> Here's a screenshot of Plannit before and after executing the command:

| Before executing the command                                   | After executing the command                                   |
|----------------------------------------------------------------|---------------------------------------------------------------|
| <p align="center"><img src="/images/add-task-before.png"/></p> | <p align="center"><img src="/images/add-task-after.png"/></p> |

<div markdown="span" class="alert alert-info">:bulb: **Tip:** 
You may view the tasks added to a module by navigating to a module's page 

> using the [`goto`](##252-navigate-between-modules) command. Alternatively, 
> you may double-click on a module to ["peek"](#224-peeking-at-modules) at a 
> module's tasks while on the home page.

Reviewer: ```suggestion
using the [`goto`](##252-navigate-between-modules) command. <br>
Alternatively, you may double-click on a module to ["peek"](#224-peeking-at-modules) at a 
module's tasks while on the home page.
```
Split into 2 sentence for user-friendliness

Example:
```
delete-task m/CS2103T tn/2
```
In the above example, we are deleting task number **2** from the module
with the module code `CS2103T`. Here's a screenshot of the command before 

> and after executing: 

Reviewer: ```suggestion
and after execution: 
```
same here  (highlighted in case it is missed out)

| <p align="center"><img src="/images/delete-task-before.png"/></p> | <p align="center"><img src="/images/delete-task-after.png"/></p> |

#### 2.2.3. Reorder tasks (swap)
You may change the order in which tasks appear in the task list of a 
particular module using the `swap-task` command.

This command will require two flags:

| Field                                                     | Flag  | Constraints                                                                                                                                                                                                         |
|-----------------------------------------------------------|-------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| **Module Code**                                           | `m/`  | Can only be non-empty string of alphanumeric characters                                                                                                                                                             |

> | **Task Numbers of Tasks whose ordering is to be swapped** | `ts/` | The two task numbers must: <ul><li>be delimited by a `SPACE` character ("` `") </li><li>correspond to a pair of existing tasks in the specified module</li> <li>be positive integers (i.e. 1, 2, 3, ... )</li></ul> |

Reviewer: Assuming <ul><li> creates a separate line, words should be caps
The two task numbers must: 
<ul><li>Be delimited by a `SPACE` character ("` `") </li>
<li>Correspond to a pair of existing tasks in the specified module</li>
 <li>Be positive integers (i.e. 1, 2, 3, ... )</li></ul>

Additionally, would the word delimited be too technical for target user?
perhaps we can consider writing
<ul><li>Be be separated with a space bar </li>

Author: I was thinking that it should not be capitalized since it is a continuation of "The two numbers must...", i.e.:
The two numbers must...
 * ...be delimited by a `SPACE` character ("` `")
 * ...correspond to a pair of existing tasks
 * ...be positive integers (i.e. 1, 2, 3, ... )

You are right regarding the technicality of "delimited" though, and I shall replace it.

swap-task m/CS2103T ts/1 3
```
In the above example, we are swapping the position of the first and third 

> task within the task list of the module with the module code `CS2103T`. The following is a

Reviewer: ```suggestion
task within the task list of the module `CS2103T`. The following is a
```

Felt that it could be confusing to read due to the
"task within the task list of the module with the ..."
perhaps we could forgo specific implementation details and go straight to the point

Author: Good point!

> screenshot of the command before and after executing. Note that ["peek"]
> (#224-peeking-at-tasks) has been enabled for clarity:

Reviewer: ```suggestion
screenshot of the command before and after execution. <br>
Note that ["peek"](#224-peeking-at-tasks) has been enabled for clarity:
```
Separated the lines for user friendliness

Author: Hmm yea was a little on the fence regarding how to format this. However I decided with the current formatting as the statement:

> The following is a screenshot of the command before and after execution.

implies that the next "item" (after the upcoming line break) is the screenshot:
> The following is a screenshot of the command before and after execution. Note, that ["peek"](#224-peeking-at-tasks) has been enabled for clarity:
>
> [SCREENSHOT]

By placing a `<br/>` before the note, the "note" becomes the next item instead:
> The following is a screenshot of the command before and after execution.
>
>Note, that ["peek"](#224-peeking-at-tasks) has been enabled for clarity:
>
> [SCREENSHOT]

Reviewer: ahh that makes sense!

### 2.2.4. Peeking at tasks
You may "peek" at a module's tasks by clicking on the module while on the 
`home` page. <br/>
 **Step 1**: Let's say you wish to view the tasks of `CS2103T SOFTWARE 
ENGINEERING` (module number 1)
<p align="center">
  <img src="/images/home-page.png" width="500"/>
</p>

**Step 2**: You will noticing a blue glow when hovering your mouse over the 
module
<p align="center">
  <img src="/images/home-page-with-cs2103-glowing.png" width="500"/>
</p>

**Step 3**: Double-clicking on the blue area will allow you to view the 
 tasks of `CS2103T SOFTWARE ENGINEERING`
<p align="center">
  <img src="/images/home-page-with-cs2103-tasks.png" width="500"/>
</p>

**Step 4**: Clicking again on the blue area will close the dropdown and hide 
tha tasks
<p align="center">
  <img src="/images/home-page-with-cs2103-glowing.png" width="500"/>
</p>

<div markdown="span" class="alert alert-info">:information_source: **Note:** 
You remain on the home page even when you "peek" at a module. To leave the 
home page and view more details of a specific module, you may use the 
[`goto`](#252-navigate-between-modules) command.

> </div>

Reviewer: Nice note!

  <img src="/images/home-page.png" width="500"/>
</p>

> **Step 2**: You will notice a blue glow when hovering your mouse over the 

Reviewer 2: Good that you spotted the grammatical error.
